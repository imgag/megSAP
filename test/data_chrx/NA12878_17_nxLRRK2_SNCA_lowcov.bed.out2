#BAM: NA12878_17.cram
#ROI: nxLRRK2_SNCA_2016_10_25.bed
#ROI regions: 2
#ROI bases: 367846
chr4	89714425	89714607	SNCA
chr4	89722305	89722335	SNCA
chr4	89755613	89755660	SNCA
chr4	89849980	89849983	
chr4	89857226	89857718	
chr4	89858007	89858435	
chr12	40210732	40210766	LINC02471, LRRK2-DT
chr12	40247505	40247749	LRRK2
chr12	40266016	40266836	LRRK2
chr12	40291243	40291367	LRRK2
chr12	40298799	40298897	LRRK2
chr12	40353065	40353298	LRRK2
chr12	40353437	40353594	LRRK2
