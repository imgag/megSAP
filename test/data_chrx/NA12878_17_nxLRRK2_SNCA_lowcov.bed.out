#BAM: NA12878_17.bam
#ROI: nxLRRK2_SNCA_2016_10_25.bed
#ROI regions: 2
#ROI bases: 367846
chr4	89714425	89714607	SNCA
chr4	89722305	89722335	SNCA
chr4	89755613	89755660	SNCA
chr4	89849981	89849983	
chr4	89857225	89857718	
chr4	89858007	89858435	
chr12	40210732	40210766	LINC02471, LRRK2-DT
chr12	40247510	40247748	LRRK2
chr12	40266018	40266041	LRRK2
chr12	40266061	40266070	LRRK2
chr12	40266071	40266836	LRRK2
chr12	40291226	40291234	LRRK2
chr12	40291237	40291367	LRRK2
chr12	40298799	40298897	LRRK2
chr12	40352664	40352688	LRRK2
chr12	40353051	40353286	LRRK2
chr12	40353429	40353603	LRRK2
